src/include/storage/data_table.h

```diff
@@ -243,28 +306,27 @@ class DataTable {
   /**
    * @return Number of blocks in the data table.
    */
-  uint32_t GetNumBlocks() const { return blocks_.size(); }
+  uint32_t GetNumBlocks() const { return blocks_size_; }
 
   /** @return Maximum number of blocks in the data table. */
   static uint32_t GetMaxBlocks() { return std::numeric_limits<uint32_t>::max(); }
 
   /**
    * @return a coarse estimation on the number of tuples in this table
    */
-  uint64_t GetNumTuple() const { return GetBlockLayout().NumSlots() * blocks_.size(); }
+  uint64_t GetNumTuple() const { return accessor_.GetBlockLayout().NumSlots() * blocks_size_; }
```

Comment:
Use the `DataTable` method rather than hardcoding the reference to `accessor_`.

Reply:
done